### PR TITLE
Support Windows in test and filter output on isWin

### PR DESCRIPTION
**Pull request by som-snytt**
As observed by sleepless Seth at https://github.com/scala/scala/pull/9185#issuecomment-689179711

### CODE REVIEW

**Comment by SethTisue** [comment]
https://scala-ci.typesafe.com/job/scala-2.13.x-integrate-windows/1501/

**Comment by som-snytt** [comment]
Hey, it worked!

Instead of the tag `internal`, we need `infernal`.

File: test/files/neg/t12134/ploogin_1.scala [hunk 1]
```diff
@@ -29,11 +29,15 @@ class Unplugged(val global: Global) extends Plugin {
...
-        import scala.util.Try
         global.settings.outdir.value = s"${global.settings.plugin.value.head}/${global.settings.outdir.value}"
         val path = Paths.get(global.settings.outdir.value)
-        val perms = asFileAttribute(EnumSet.of(OWNER_READ))
-        Try(createFile(path, perms))
+        val file = path.toFile()
+        //import scala.util.Try
+        //val perms = asFileAttribute(EnumSet.of(OWNER_READ))
+        //val res = Try(createFile(path, perms))
+        //assert(res.isSuccess)
```
Comment (dwijnand) [inline test/files/neg/t12134/ploogin_1.scala:38]: Lose the comment code or justify it, IMO

**Review by dwijnand — APPROVED** [review]
LGTM.  Dropping the commented code would be nice, but it's also just a test file..

File: test/files/neg/t12134/ploogin_1.scala (hunk 1 above)
Comment (som-snytt) [reply test/files/neg/t12134/ploogin_1.scala:38]: I wished to preserve my hours of researches into file creation on the JVM.

**Pull request merged**